<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP客户端</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,2,5,1">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>域名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="domainEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,5,3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>端口号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,5">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,2,5,1">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>登录名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,5,3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>口令：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pwEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,13">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>文件目录列表：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="qButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="upButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>上一层</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ulButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>上传</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>下载</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="eButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,13">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>远程目录列表：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
